docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -116,37 +122,37 @@ public static PredictionModel<TaxiTrip, TaxiTripFarePrediction> Train()
 }
 ```
 
-## Create a learning pipeline
-
-The learning pipeline loads all of the data and algorithms necessary to train the model. Add the following code into the `Train()` method:
+The learning pipeline loads all of the data and algorithms necessary to train the model. Add the following code into the `Train` method:
 
 ```csharp
 var pipeline = new LearningPipeline();
 ```
 
-## Load and transform your data
+## Load and transform data
 
-Next, load your data into the pipeline. Point to the `_datapath` created initially and specify the delimiter of the .csv file (,). Add the following code into the `Train()` method underneath the last step:
+The first step that the learning pipeline performs is loading data from the training data set. In our case, training data set is stored in the text file, which path is defined by the value of the `_datapath` constant. That file contains the header with the column names, so the first row should be ignored while loading data. Columns in the file are separated by the comma (","). Add the following code into the `Train` method:
 
 ```csharp
-pipeline.Add(new TextLoader(_datapath).CreateFrom<TaxiTrip>(separator:','));
+pipeline.Add(new TextLoader(_datapath).CreateFrom<TaxiTrip>(useHeader: true, separator: ','));
 ```
 
-You'll refer to the columns without the underscores in the code you're creating. Copy the `FareAmount` column into a new column called "Label" using the `ColumnCopier()` function. This column is the **Label**.
+In the next steps we refer to the columns by the names defined in the `TaxiTrip` class.
+
+When the model is trained, the values in the column named **Label** are considered as values to be predicted. As we want to predict the taxi trip fare, copy the `FareAmount` column into the **Label** column. To do that, use <xref:Microsoft.ML.Transforms.ColumnCopier> and add the following code:
```

Comment:
@aditidugar please check the updated wording about the **Label** column, which is another special column in ML.NET.

Reply:
Yes or have one of your fields called Label